jobs/kpi-forecasting/kpi_forecasting/tests/test_metric_hub.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+
+from pandas import to_datetime
+from kpi_forecasting.metric_hub import MetricHub
+
+
+def test_metrichub_for_dau_kpi():
+    test_metric_hub = MetricHub(
+        app_name="multi_product",
+        slug="mobile_daily_active_users_v1",
+        start_date="2024-01-01",
+    )
+    now = to_datetime(datetime.utcnow()).date()
+
+    query = test_metric_hub.query()
+    query_where = f"WHERE submission_date BETWEEN '2024-01-01' AND '{now}'\n GROUP BY"
+
+    assert query_where in query
```

Comment:
I love that you added tests!!
Would it be better to hardcode the entire expected query in order to make sure the full query is returned as expected?

Reply:
I thought about that, too, and I don't think so, as this is the only piece of the query that would change if a WHERE clause is supplied. Also, the tests start looking really gross with all of those SQL strings hanging around.